cmd/kubectl-gadget/bcck8s/bcck8s.go

```diff
@@ -112,24 +60,16 @@ func bccCmd(subCommand, bccScript string) func(*cobra.Command, []string) error {
 
 		extraParams := ""
 
-		// disable manager for biotop
-		if subCommand == "biotop" {
-			extraParams += " --nomanager"
-		}
-
 		gadgetParams := ""
 
 		// add container info to gadgets that support it
-		if subCommand != "tcptop" && subCommand != "profile" {
+		if subCommand != "profile" {
 			gadgetParams = "--containersmap /sys/fs/bpf/gadget/containers"
 		}
 
-		var transform func(line string) string
-
 		if params.OutputMode == utils.OutputModeCustomColumns {
```

Comment:
>Given all this. What do you think about my proposal to implement the golang control plane of the profile gadget before this PR?

We already discuss this, and I would have really been happy to have all the gadgets translated to golang control plane in the next release.
But, this PR is opened since 2 months and right now I just want it to be merged as it is.